CHANGELOG.develop

```diff
@@ -694,6 +707,12 @@ Benner and Paweł Siudak):
   Benner)
 - Fixed ~TAB~ to toggle occurrences in iedit state in Vim and Hybrid editing
   styles (thanks to Sylvain Benner)
+- Fix the projectile layer (thanks to Karol Wójcik)
+- Fix handling of keymap property when the value is a symbol instead of
+  a keymap (thanks to Rudi Grinberg)
+- Load =evil-surround= when entering evil's operator state to guarantee it
+  is available for =evil-yank= and =evil-delete= (thanks to bmag)
```

Comment:
This is a fix to the lazy loading feature added since the last release, so I'd just add bmag to the "thanks to" for the existing entry ("Extensive use of lazy loading of packages [...]").